nncf/experimental/openvino_native/quantization/algorithms/fast_bias_correction/ov_backend.py

```diff
@@ -0,0 +1,167 @@
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from collections import deque
+from typing import Dict, List, Tuple, Optional
+
+import numpy as np
+import openvino.runtime as ov
+
+from nncf.common.graph import NNCFGraph, NNCFNode
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.tensor_statistics.collectors import ReductionShape
+from nncf.common.utils.backend import BackendType
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVOpMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import LAYERS_WITH_BIAS_METATYPES
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVConvolutionMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVConvolutionBackpropDataMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVMatMulMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVFakeQuantizeMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVConvertMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVConstantMetatype
+from nncf.experimental.openvino_native.graph.metatypes.openvino_metatypes import OVAddMetatype
+from nncf.experimental.openvino_native.graph.model_transformer import OVModelTransformer
+from nncf.experimental.openvino_native.graph.transformations.commands import OVBiasCorrectionCommand
+from nncf.experimental.openvino_native.graph.transformations.commands import OVModelExtractionCommand
+from nncf.experimental.openvino_native.graph.transformations.commands import OVTargetPoint
+from nncf.experimental.openvino_native.statistics.collectors import OVMeanStatisticCollector
+from nncf.experimental.openvino_native.statistics.collectors import OVNNCFCollectorTensorProcessor
+from nncf.experimental.openvino_native.tensor import OVNNCFTensor
+from nncf.quantization.algorithms.fast_bias_correction.backend import ALGO_BACKENDS
+from nncf.quantization.algorithms.fast_bias_correction.backend import FastBiasCorrectionAlgoBackend
+
+
+@ALGO_BACKENDS.register(BackendType.OPENVINO)
+class OVFastBiasCorrectionAlgoBackend(FastBiasCorrectionAlgoBackend):
+
+    @property
+    def layers_with_bias_metatypes(self) -> List[OVOpMetatype]:
+        return LAYERS_WITH_BIAS_METATYPES
+
+    @property
+    def channel_axis_by_types(self) -> Dict[OVOpMetatype, int]:
+        return {
+            OVConvolutionMetatype: 1,
+            OVConvolutionBackpropDataMetatype: 1,
+            OVMatMulMetatype: -1
+        }
+
+    @property
+    def tensor_processor(self) -> OVNNCFCollectorTensorProcessor:
+        return OVNNCFCollectorTensorProcessor()
+
+    @staticmethod
+    def model_transformer(model: ov.Model) -> OVModelTransformer:
+        return OVModelTransformer(model)
```

Comment:
```suggestion
```
We should use https://github.com/openvinotoolkit/nncf/blob/abc05b679701fea8d318b8fadbab70a1ca53756a/nncf/common/factory.py#L45